src/modules/settings/game_settings.gd

```diff
@@ -140,7 +140,7 @@ func rebind_request(
 	action: String, old_key: InputEventKey, new_key: InputEventKey, action_id
 ) -> bool:
 	var success := true
-	var old_keycode = old_key.keycode
+	#var old_keycode = old_key.keycode
```

Comment:
Big proponent of deleting code and not deleting it

Reply:
Yeah I would also like to delete it, but let me check first then.

@PlantDev can I delete this or do you have plans for this variable?